src/tron.jl

```diff
@@ -217,6 +217,10 @@ function SolverCore.solve!(
     error("tron should only be called for unconstrained or bound-constrained problems")
   end
 
+  if(verbose > 0)
```

Comment:
Ok, but we need to show this warning if `verbose > 0` AND the initial guess provided is not within bounds.